tests/test_lora.py

```diff
@@ -279,3 +282,20 @@ def test_lora_qkv_linear_weights_merged_status(rank, enable_lora, expected_merge
     assert not layer.merged
     layer.merge()
     assert layer.merged == expected_merged
+
+
+def test_bnb_replacement():
+    from quantize.bnb import _BITSANDBYTES_AVAILABLE
+
+    if not _BITSANDBYTES_AVAILABLE:
+        pytest.skip("BNB not available")
+
+    from quantize.bnb import bnb
+    from lit_gpt.utils import quantization
+    from lit_gpt.lora import LoRAQKVLinear, LoRALinear
+
+    with quantization("bnb.nf4"):
```

Comment:
Maybe a dumb question but shouldn't this test also check other precision types from bnb?
`bnb.int8`, `bnb.fp4` and maybe their variants with double quantization?

Reply:
Done